Software/src/AbstractLedDevice.hpp

```diff
@@ -85,11 +85,15 @@ public slots:
       \obsolete only form compatibility with Lightpack ver.<=5.5 hardware
      \param value bits per channel
     */
-    virtual void setColorDepth(int value) = 0;
+	virtual void setColorDepth(int);
 
 protected:
+	virtual void setColorsUnsmoothed(const QList<QRgb> & colors) = 0;
```

Comment:
Yes. This thing is not merge-ready. It is a PR so my CI builds it automatically for testing, but this approach seems to be too much for the device to handle. Additionally, the smoothing algorithm (which I took from the LP firmware) behaves differently when the number of updates is different. I'm still looking for a good solution